pages/index.vue

```diff
@@ -13,23 +13,33 @@
     />
     <section class="container">
       <div class="mt-3 w-100">
-        <ContentCardSection
-          v-for="section in page.hasPart"
-          :key="section.sys.id"
-          :section="section"
-        />
+        <template v-if="isRichText">
```

Comment:
A page may contain multiple sections of different types. This therefore needs to be beneath the `v-for` on `page.hasPart` and test the type of each section.

Reply:
So you want the Rich text component in the contentCardSection component?

Reply:
No.

We need something like:
```html
<template
  v-for="section in page.hasPart"
  :key="section.sys.id"
>
  <RichText
    v-if="isRichText"
    :headline="section.fields.headline"
    :text="section.fields.text"
  />
  <ContentCardSection
    v-else-if="isCardGroup"
    :section="section"
  />
</template>
```

Reply:
Thank you Rich. I now follow